### PR TITLE
Modify rule S6255: Add new Terraform code sample

### PR DESCRIPTION
**Pull request by hendrik-buchwald-sonarsource**
Implementation ticket: https://sonarsource.atlassian.net/browse/SONARIAC-416

### CODE REVIEW

File: rules/S6255/terraform/rule.adoc [hunk 1]
```diff
@@ -50,6 +65,24 @@ resource "aws_s3_bucket" "example" { # Compliant
...
+  bucket = "example"
+}
+
+resource "aws_s3_bucket_versioning" "example" {
+  bucket = aws_s3_bucket.example.id
+  versioning_configuration {
+    status = "Enabled"
+    mfa_delete = "Enabled"
+  }
+  mfa = "${var.MFA}"
```
Comment (loris-s-sonarsource) [inline rules/S6255/terraform/rule.adoc:82]: minor - we should replace the variable with "true" here for clarity

**Review by loris-s-sonarsource — APPROVED** [review]
LGTM -- 1 minor suggestion

File: rules/S6255/terraform/rule.adoc (hunk 1 above)
Comment (hendrik-buchwald-sonarsource) [reply rules/S6255/terraform/rule.adoc:82]: I am not sure I understand what you mean, could you clarify?

Comment (loris-s-sonarsource) [reply rules/S6255/terraform/rule.adoc:82]: I didn't understand what the mfa variable was about, so I assumed it was a boolean. But no, it's the actual MFA code, right ?

Comment (hendrik-buchwald-sonarsource) [reply rules/S6255/terraform/rule.adoc:82]: Ah, yea, it is the actual MFA code. I could specify a fake one but I did not want to show a hard-coded MFA token as an example.

Comment (loris-s-sonarsource) [reply rules/S6255/terraform/rule.adoc:82]: Ok, got it. My train of thought was that using a variable could be ambiguous, but it looks like it doesn't matter here. We can dismiss my comment

**Pull request merged**